var/spack/repos/builtin/packages/graphviz/package.py

```diff
@@ -92,7 +94,7 @@ class Graphviz(AutotoolsPackage):
     depends_on('glib', when='+pangocairo')
     depends_on('libpng', when='+pangocairo')
     depends_on('pango', when='+pangocairo')
-    depends_on('zlib', when='+pangocairo')
+    depends_on('zlib')
```

Comment:
Tried building `~pangocairo` but it still picked up the system zlib, so I guess it's always required?